<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="labelID">
      <property name="text">
       <string>学号:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAge">
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="lineEditID"/>
     <widget class="QLineEdit" name="lineEditName"/>
     <widget class="QLineEdit" name="lineEditAge"/>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditID</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditAge</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slotEnsure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>slotReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotEnsure()</slot>
  <slot>slotReset()</slot>
 </slots>
</ui>
